components/HearingsScheduled/HearingsScheduled.module.css

```diff
@@ -0,0 +1,117 @@
+.heading {
+  color: var(--bs-blue);
+  font-size: 4em;
```

Comment:
I have always used `rem` instead of `em` as it doesn't depend on the font size of the surrounding elements, so it's a bit easier to reason about. Though I don't know what the best practice is. What do you think?

Reply:
Typically, I always use rem for font size. Margins & padding (especially padding) I usually use em as this allows for the adjusting of the font size and having the spacing around it adjust with it proportionally automatically allowing fewer media query adjustments. (The exception being in places you know for certain that the amount of space should never change despite font size- then rem) For media queries, I always use ems as it's the only proportional unit that is consistent across browsers (Rem is calculated differently in media queries in Safari vs Chrome).
For most, these are considered best practices but it can be a controversial issue. 
I'm happy to convert margins & padding to only rem (except media queries) if you want me to for simplicity. If we change any font sizes later, it might mean hand-tuning all the padding.

Reply:
Thanks for explaining this to me, Kep! LGTM